Controller/HelperController.php

```diff
@@ -110,7 +110,7 @@ public function appendFormFieldElementAction(Request $request)
         // render the widget
         // todo : fix this, the twig environment variable is not set inside the extension ...
 
-        $extension = $this->twig->getExtension('form');
+        $extension = $this->twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension');
```

Comment:
I'm away for a few days so it will be hard to help, ping me at the end of next week if you still need to please